build_deploy.sh

```diff
@@ -99,4 +103,31 @@ make \
   internal_image_registry="quay.io" \
   image_repository="$QUAY_ORG/kas-fleet-manager" \
   docker/login \
-  image/push
\ No newline at end of file
+  image/push
+
+# create new relaase in managed-kafka-versions repo for kas-fleet-manager
+if [[ ! -z "$AUTHOR_EMAIL" ]] && [[ ! -z "$AUTHOR_NAME" ]] && [[ ! -z "$GITLAB_TOKEN" ]] && \
+[[ ! -z "$PASSWORD" ]] && [[ ! -z "$USERNAME" ]]; then
+  LATEST_COMMIT=$(git rev-parse HEAD)
+  echo "$LATEST_COMMIT"
+  SECOND_LAST_COMMIT=$(git rev-parse HEAD~1)
+  echo "$SECOND_LAST_COMMIT"
+  IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+  echo "$IMAGE_TAG"
```

Comment:
these echo statements could potentially be removed. Interested to hear how much do we want to log (if anything)

Reply:
If we want to log them I thing we should at least add a label to make the log readable:
```bash
  LATEST_COMMIT=$(git rev-parse HEAD)
  echo "LATEST COMMIT: $LATEST_COMMIT"
  SECOND_LAST_COMMIT=$(git rev-parse HEAD~1)
  echo "SECOND-LAST COMMIT:  $SECOND_LAST_COMMIT"
  IMAGE_TAG=$(git rev-parse --short=7 HEAD)
  echo "IMAGE TAG: $IMAGE_TAG"

```